example/core_tutorial/13_hdf5_output/hdf5_output_example.cpp

```diff
@@ -0,0 +1,180 @@
+    double grid_max[3] = { 3.0, 3.0, 3.0 };
+    double grid_delta[3] = { 1.0, 1.0, 1.0 };
+
+    /*
+      Create the particle ids.
+    */
+
+    /*
+      Get the particle ids, coordinates, velocity, radius
+    */
+    auto ids = Cabana::slice<1>( aosoa, "ids" );
+    auto positions = Cabana::slice<0>( aosoa, "positions" );
+    auto velocity = Cabana::slice<2>( aosoa, "velocity" );
+    auto radius = Cabana::slice<3>( aosoa, "radius" );
+
+    // initialize the particle properties
+    for ( std::size_t i = 0; i < aosoa.size(); ++i )
+    {
+
+        // set ids of the particles
+        ids( i ) = i;
+
+        // set positions of the particles
+        positions( i, 0 ) = grid_min[0] + grid_delta[0] * ( 2. * i );
+        positions( i, 1 ) = 0.;
+        positions( i, 2 ) = 0.;
+
+        // set the velocity of each particle
+        velocity( i, 0 ) = 1.;
+        velocity( i, 1 ) = 1.;
+        velocity( i, 2 ) = 0.;
+
+        // set the radius of each particle
+        radius( i ) = 0.1;
+    }
+
+    /*
+      We will evolve the system for a total time of 1000 timesteps. We will
+      output the properties at an interval of every 100 steps.
+    */
+    auto dt = 1e-2;       // timestep
+    auto tf = 1000. * dt; // total time
+    auto t = 0.;          // time
+    int nsteps = tf / dt; // total steps
+    int pfreq = 100;      // print frequency
+    // Main timestep loop
+    for ( int step = 0; step < nsteps; step++ )
+    {
+        /*
+          move positions
+        */
+        Kokkos::RangePolicy<ExecutionSpace> policy( 0, aosoa.size() );
+        Kokkos::parallel_for(
+            "move_particles", policy, KOKKOS_LAMBDA( const int i ) {
+                positions( i, 0 ) += velocity( i, 0 ) * dt;
+                positions( i, 1 ) += velocity( i, 0 ) * dt;
+                positions( i, 2 ) += velocity( i, 0 ) * dt;
+            } );
+        Kokkos::fence();
+
+        // This is for setting HDF5 options
+        Cabana::Experimental::HDF5ParticleOutput::HDF5Config h5_config;
+        if ( step % pfreq == 0 )
+        {
+
+            /*
+              Inorder to write the particle properties to an hdf5 files, we use
+
+              Cabana::Experimental::HDF5ParticleOutput::writeTimeStep function.
+
+              First seven arguments of `writeTimeStep` are mandotory arguments
+              and rest are optional fields. The seven arguments are as follows
+
+
+              1. config file
+              2. name of the file
+              3. MPI communication
+              4. time step value (int)
+              5. time of the simulation (double)
+              6. number of particles (int)
+              7. positions (double[3])
+              8. Optional fields
+              9. Optional fields
+              10. ...
+
+              Please look at the source for more details
+              https://github.com/ECP-copa/Cabana/blob/master/core/src/Cabana_HDF5ParticleOutput.hpp#L496
```

Comment:
Maybe refer to the documentation instead:

https://ecp-copa.github.io/Cabana/doxygen/Cabana__HDF5ParticleOutput_8hpp.html

Reply:
I removed for now, but I am going to look for a way to automatically link each example to the doxygen